pkg/mcs/utils/util.go

```diff
@@ -45,30 +47,26 @@ import (
 	"google.golang.org/grpc/keepalive"
 )
 
-const (
-	// maxRetryTimes is the max retry times for initializing the cluster ID.
-	maxRetryTimes = 5
-	// ClusterIDPath is the path to store cluster id
-	ClusterIDPath = "/pd/cluster_id"
-	// retryInterval is the interval to retry.
-	retryInterval = time.Second
-)
-
 // InitClusterID initializes the cluster ID.
 func InitClusterID(ctx context.Context, client *clientv3.Client) (id uint64, err error) {
-	ticker := time.NewTicker(retryInterval)
+	ticker := time.NewTicker(constant.RetryInterval)
 	defer ticker.Stop()
-	for i := 0; i < maxRetryTimes; i++ {
-		if clusterID, err := etcdutil.GetClusterID(client, ClusterIDPath); err == nil && clusterID != 0 {
+	retryTimes := 0
+	for {
+		if clusterID, err := etcdutil.GetClusterID(client, constant.ClusterIDPath); err == nil && clusterID != 0 {
 			return clusterID, nil
 		}
 		select {
 		case <-ctx.Done():
 			return 0, err
 		case <-ticker.C:
+			retryTimes++
```

Comment:
It will print the log every 250s until it succeeds?

Reply:
same as previous